scripts/filter_markets.py

```
@@ -0,0 +1,1350 @@
+                    k32.CloseHandle(handle)
+                    return True
+                return False
+        else:
+            os.kill(pid, 0)
+            return True
+    except (OSError, ProcessLookupError, ValueError):
+        return False
+
+
+def q_min(a: float, b: float) -> float:
+    return max(min(a, b), max(a / C, b / C))
```

Ruff (0.16.2)

[warning] Nested `max` calls can be flattened

Flatten nested `max` calls

(PLW3301)